umap/views.py

```diff
@@ -622,6 +611,36 @@ def post(self, *args, **kwargs):
         return simple_json_response()
 
 
+class SendEditLink(FormLessEditMixin, FormView):
+    form_class = SendLinkForm
+
+    def post(self, form, **kwargs):
+        self.object = kwargs["map_inst"]
+        if (
+            self.object.owner
+            or not self.object.is_anonymous_owner(self.request)
+            or not self.object.can_edit(self.request.user, self.request)
+        ):
+            return HttpResponseForbidden()
+        form = self.get_form()
+        if form.is_valid():
+            email = form.cleaned_data["email"]
+        else:
+            return HttpResponseBadRequest("Invalid")
+        link = self.object.get_anonymous_edit_url()
+
+        send_mail(
+            _("Your secret edit link"),
```

Comment:
```suggestion
            _("The uMap edit link for your map: %(map_name)s" % {"map_name": self.object.name}),
```

This original subject looks spam-y to me :)